DESCRIPTION

```diff
@@ -2,6 +2,14 @@ Package: finalsize
 Title: Calculates final size for SIR epidemic in heterogenous population
 Version: 0.1
 Authors@R: person("Adam", "Kucharski", , "adam.kucharski@lshtm.ac.uk", role = c("aut", "cre"), comment = c(ORCID = "0000-0001-8814-9421"))
```

Comment:
> Thanks for the PR - very nice work! Since this is an initial setup, feel free to leave some of these comments / suggestions out for a later iteration.

Thanks - I'm also unsure how far to take it especially since I'm not sure this is the package's final form or even close to it. I'll implement the current suggestions now, but a roadmap for the package would be good to have. See also #21 where @sbfnk suggests other functionality that could be added (although would that be a duplication?).

Reply:
Could we make this a standing issue, so all new contributors identify themselves in the issue thread and can be added?